view/frontend/web/css/pagseguro.css

```diff
@@ -1,6 +1,9 @@
 .checkout-payment-method .input-text.ps_cc_birthdate, .checkout-payment-method .input-text.cc-exp-date {
 	width: 50px !important;
 }
+.checkout-payment-method .input-text.ps_cc_birthdate_year {
+    width: 55px !important;
```

Comment:
Although we use !important in the above rule, we should avoid it in future releases. It will downgrade our code quality (which is already F for this file) and make it harder for others to extend it. 
See more at https://app.codacy.com/gh/r-martins/PagSeguro-Magento-Transparente-M2/file/51372120862/issues/source?bid=16868127&fileBranchId=16868127.